shared/common-adapters/name-with-icon/index.stories.js

```diff
@@ -23,34 +22,63 @@ const provider = Sb.createPropProviderWithCommon(
 const load = () => {
   Sb.storiesOf('Common/NameWithIcon', module)
     .addDecorator(provider)
-    .addDecorator(story => <ScrollView>{story()}</ScrollView>)
+    .addDecorator(Sb.scrollViewDecorator)
     .add('Vertical', () => (
       <React.Fragment>
-        <NameWithIcon {...commonProps} username="ayoubd" isYou={true} metaOne="Danny Ayoub" size="default" />
+        <NameWithIcon
+          {...commonProps}
+          icon="icon-computer-96"
+          title="Home laptop"
+          metaOne="Current device"
+          metaTwo="Computer"
+          size="big"
+        />
+        <NameWithIcon {...commonProps} icon="icon-phone-48" title="iPhone 8" metaTwo="Phone" size="small" />
         <NameWithIcon
           {...commonProps}
           teamname="keybasefriends"
           title="keybasefriends"
           metaOne="786 members"
-          size="default"
+          metaTwo="TEAM"
+          size="big"
+        />
+        <NameWithIcon
+          {...commonProps}
+          teamname="keybasefriends"
+          title="keybasefriends"
+          metaOne="786 members"
+          size="small"
         />
-        <NameWithIcon {...commonProps} username="cecileb" metaOne="Cécile Boucheron" size="small" />
         <NameWithIcon
           {...commonProps}
-          icon="iconfont-crown-admin"
```

Comment:
no need for an example with the admin anymore?

Reply:
Nah, we never really use the component with an icon from the icon font.